packages/app/src/display/style/components/ui/_overlay.scss

```diff
@@ -153,6 +161,26 @@
 		width: 1em;
 		height: 0.625em;
 	}
+
+	.quick-chat-box-container {
+		display: flex;
+		flex-direction: column;
+		width: 100%;
+		align-items: center;
+		justify-content: center;
```

Comment:
Agree with your comments here about only having one centre.

If you use chrome dev tools you should be able to turn these off individually and see which ones are important